stdlib/number.gr

```diff
@@ -493,3 +952,107 @@ export let cos = (radians: Number) => {
     sin(pi / 2 + radians)
   }
 }
+
+/**
+ * Computes the tangent of a number (in radians) using Chebyshev polynomials.
+ *
+ * @param radians: The input in radians
+ * @returns The computed tangent
+ *
+ * @since v0.5.4
+ */
+export let tan = (radians: Number) => {
+  if (isNaN(radians) || isInfinite(radians)) {
+    nan
+  } else {
+    sin(radians) / cos(radians)
+  }
+}
+
+// Math.gamma implemented using the stirling's approximation
+// https://en.wikipedia.org/wiki/Lanczos_approximation
+/**
+ * Computes the gamma function of a value using
+ *
+ * @param z: The value to compute the gamma of
```

Comment:
```suggestion
 * @param z: The value to factorialize
```
Is this correct? I just quickly skimmed wikipedia

Reply:
This is not correct the gamma function is its own thing it just happens to work as a good approximation for gamma in the case of decimals. There are a bunch of differences though between gamma and factorial related to valid ranges and a few other differences.

Reply:
What should we write here? @ospencer

Reply:
The not-ending-in-a-preposition is something I really struggle with; I don't have anything better than "The value of which to compute gamma".